### PR TITLE
Multiselect

### PR DESCRIPTION
**Pull request by Ignalion**
#199 

Hi, @AllenFang

This is my draft implementation of multi select. I think it works pretty stratiforward although It might cause some issues while using with remote bc I didn't test it.

Please take a look if it's ok with your thoughts.

P.S. I rebased my branch to develop so had to recreate PR

### CODE REVIEW

**Comment by Ignalion** [comment]
Hi, @AllenFang 

Actually I have a couple more ideas about custom filters so could you please review the PR to say whether it's OK from your perspective? I'd like to implement range filter as well, I think the same way.

**Comment by AllenFang** [comment]
@Ignalion sure, sorry I'm a little busy recently, will have a look in this couple day, thanks

**Comment by AmitKKhanchandani** [comment]
@AllenFang is it possible to merge this today?

Currently, my project needs a multifilter select functionality, any help/idea how would I add achieve this?

File: packages/react-bootstrap-table2-example/package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "react-bootstrap-table2-example",
-  "version": "0.1.10",
+  "version": "0.1.11",
```
Comment (AllenFang) [inline packages/react-bootstrap-table2-example/package.json]: Don''t patch version manually, we use `lerna`. Please remove this changes

**Comment by AllenFang** [comment]
@AmitKKhanchandani sorry, I think I still need some works on this PR, if ok, probably able to release in this weekend.

**Comment by AllenFang** [comment]
@Ignalion I think you have to rebase your branch firstly, you contain a wrong commit in this patch. If you don't know how to do, you can 
1. recreate a new branch 
2. cherry pick your patch from current branch to new branch
3. submit a new PR.

File: packages/react-bootstrap-table2-example/package.json (hunk 1 above)
Comment (Ignalion) [reply packages/react-bootstrap-table2-example/package.json]: Actually, I just rebased branch to develop, i.e. i didn't patch it manually. it came from develop branch, I think

**Comment by Ignalion** [comment]
I rebased branch from develop and re-created PR #438

**Comment by AllenFang** [comment]
released on `react-bootstrap-table2-filter@0.3.2`

**Pull request closed without merging**